README.md

```diff
@@ -92,14 +92,21 @@ Your `~/.vimrc.local` might look like this:
 If you don't wish to install a vim plugin from the default set of vim plugins in
 `.vimrc.bundles`, you can ignore the plugin by calling it out with `UnPlug` in
 your `~/.vimrc.bundles.local`.
-blacklist in `~/.vimrc.ignorebundles`:
 
     " Don't install vim-scripts/tComment
     UnPlug 'tComment'
 
 `UnPlug` can be used to install your own fork of a plugin or to install a shared
 plugin with different custom options.
 
+    " Only load vim-coffee-script if a Coffeescript buffer is created
+    UnPlug 'vim-coffee-script'
+    Plug 'kchmck/vim-coffee-script', { 'for': 'coffee' }
```

Comment:
I've been doing this for several of the plugins I load from my personal config and the mechanism seems to work without problems.

The plugin that really affects startup time is `rails.vim`, and there's good reason to have that load even if you're not opening a Ruby buffer. We could do this for some of the plugins, but I don't think we'd get a huge performance benefit.